src/azure-cli/azure/cli/command_modules/network/tests/latest/test_private_endpoint_commands.py

```diff
@@ -672,82 +672,81 @@ def test_private_link_resource_batch_account(self, resource_group, batch_account
-                 '--disable-private-endpoint-network-policies true',
-                 checks=self.check('privateEndpointNetworkPolicies', 'Disabled'))
-
-        # Create a private endpoint connection
-        pe = self.cmd('network private-endpoint create -g {rg} -n {pe} --vnet-name {vnet} --subnet {subnet} -l {loc} '
-                      '--connection-name {pe_connection} --private-connection-resource-id {acc_id} '
-                      '--group-id Sql').get_output_in_json()
-        self.kwargs['pe_id'] = pe['id']
-        self.kwargs['pe_name'] = self.kwargs['pe_id'].split('/')[-1]
-
-        # Show the connection at cosmos db side
-        results = self.kwargs['pe_id'].split('/')
-        self.kwargs[
-            'pec_id'] = '/subscriptions/{0}/resourceGroups/{1}/providers/Microsoft.DocumentDB/databaseAccounts/{2}/privateEndpointConnections/{3}'.format(
-            results[2], results[4], self.kwargs['acc'], results[-1])
-        self.cmd('network private-endpoint-connection show --id {pec_id}',
-                 checks=self.check('id', '{pec_id}'))
-        self.cmd(
-            'network private-endpoint-connection show --resource-name {acc} --name {pe_name} --resource-group {rg} --type Microsoft.DocumentDB/databaseAccounts',
-            checks=self.check('name', '{pe_name}'))
-        self.cmd('network private-endpoint-connection show --resource-name {acc} -n {pe_name} -g {rg} --type Microsoft.DocumentDB/databaseAccounts',
-                 checks=self.check('name', '{pe_name}'))
-
-        # Test approval/rejection
-        self.kwargs.update({
-            'approval_desc': 'You are approved!',
-            'rejection_desc': 'You are rejected!'
-        })
-        self.cmd(
-            'network private-endpoint-connection approve --resource-name {acc} --resource-group {rg} --name {pe_name} --type Microsoft.DocumentDB/databaseAccounts '
-            '--description "{approval_desc}"', checks=[
-                self.check('properties.privateLinkServiceConnectionState.status', 'Approved'),
-                self.check('properties.privateLinkServiceConnectionState.description', '{approval_desc}')
-            ])
-        self.cmd('network private-endpoint-connection reject --id {pec_id} '
-                 '--description "{rejection_desc}"',
-                 checks=[
-                     self.check('properties.privateLinkServiceConnectionState.status', 'Rejected'),
-                     self.check('properties.privateLinkServiceConnectionState.description', '{rejection_desc}')
-                 ])
-        self.cmd('network private-endpoint-connection list --name {acc} --resource-group {rg} --type Microsoft.DocumentDB/databaseAccounts', checks=[
-            self.check('length(@)', 1)
-        ])
-
-        # Test delete
-        self.cmd('network private-endpoint-connection delete --id {pec_id} -y')
+# To be uncommented when https://github.com/Azure/azure-cli/issues/14194 is resolved
```

Comment:
Why do we disable all test for private link?

Reply:
These also require Network module. Would add the tests back when https://github.com/Azure/azure-cli/issues/14194 is resolved.

Reply:
it is already resolved.

Reply:
please add all test back because it seems that the issue is already resolved.